.github/workflows/nox.yml

```
@@ -22,35 +22,37 @@ env:
 jobs:
   nox:
     runs-on: ubuntu-latest
-    defaults:
-      run:
-        working-directory: antsibull-build
     strategy:
       fail-fast: false
       matrix:
         include:
           - session: test
             python-versions: "3.9, 3.10, 3.11, 3.12, 3.13"
+            force-python: ""
             codecov: true
             packages: ""
 
           - session: "check_package_files(8.1.0)"
```

```suggestion
          - session: "check_package_files-3.13(8.1.0)"
```

Appending the Python version here (and for the other ones) seems to workaround the problem.

---

I've implemented that (and fixed some Python versions) in e721edfe6cad2702ae99ed7cf988d673d7ccf2c6.